sdk/resourcemanager/Azure.ResourceManager.Core/src/ResourceGroupContainer.cs

```diff
@@ -219,9 +219,6 @@ public virtual AsyncPageable<ResourceGroup> ListAsync(CancellationToken cancella
         /// <inheritdoc />
         public override ArmResponse<ResourceGroup> Get(string resourceGroupName, CancellationToken cancellationToken = default)
         {
-            if (string.IsNullOrWhiteSpace(resourceGroupName))
```

Comment:
don't we still need to do this validation?

Reply:
this validation happens inside the restClient and that is by design so that the diagnostics scope can capture the faults here.  The RestClient only checks null (which we can discuss tweaking) but by design the error handling happens in the rest client